server/sonar-server/src/main/java/org/sonar/server/property/InternalProperties.java

```diff
@@ -32,6 +32,8 @@
    */
   String DEFAULT_ORGANIZATION = "organization.default";
 
+  String ORGANIZATION_ENABLED = "organization.enabled";
```

Comment:
how come this constant is added in this commit when it's already used in commit "SONAR-8761 add class OrganizationFeature" two commits before?